### ISSUE TITLE
How to finetune CogVideoX1.5-5B T2V LoRA?

### ISSUE THREAD

**Issue by MaulanaYusufIkhsanRobbani** [a-r-r-o-w/finetrainers#378]
Hello. I still unfamiliar with the finetuning process. I want to finetune CogVideoX1.5-5B T2V with LoRA. I have single RTX 4090. I try to re-run the bash script "finetrainers\examples\training\sft\cogvideox\crush_smol_lora\train.sh" with my own dataset and end up with error message
`train.sh: line 130: accelerate: command not found
train.sh: line 131: $'(\r --parallel_backend accelerate\r --pp_degree 1 --dp_degree 1 --dp_shards 1 --cp_degree 1 --tp_degree 1\r\r)\r': command not found
: No such file or directory_path THUDM/CogVideoX1.5-5B
 --dataset_config D:/TA_ucup/finetrainers/examples/training/sft/cogvideox/crush_smol_: No such file or directoryize 10
train.sh: line 134: $'(\r --dataloader_num_workers 0\r)\r': command not found
train.sh: line 135: $'(\r --flow_weighting_scheme logit_normal\r)\r': command not found
train.sh: line 136: $'(\r --training_type lora\r --seed 42\r --batch_size 1\r --train_steps 3000\r --rank 32\r --lora_alpha 32\r --target_modules (transformer_blocks|single_transformer_blocks).*(to_q|to_k|to_v|to_out.0)\r --gradient_accumulation_steps 1\r --gradient_checkpointing\r --checkpointing_steps 1000\r --checkpointing_limit 2\r --enable_slicing\r --enable_tiling\r)\r': command not found
train.sh: line 137: $'(\r --optimizer adamw\r --lr 5e-5\r --lr_scheduler constant_with_warmup\r --lr_warmup_steps 1000\r --lr_num_cycles 1\r --beta1 0.9\r --beta2 0.99\r --weight_decay 1e-4\r --epsilon 1e-8\r --max_grad_norm 1.0\r)\r': command not found
 --validation_dataset_file D:/TA_ucup/finetrainers/examples/training/sft/cogvideox/cr: No such file or directoryon
: No such file or directoryogvideoxeox`
I already install the library requirements and the diffusers. Is there anything I missing?

**Comment by a-r-r-o-w** [comment]
Are you on Windows or Linux? If on Windows, do you have WSL installed?

**Comment by MaulanaYusufIkhsanRobbani** [comment]
I use windows. I tried with wsl still found the same error

**Comment by a-r-r-o-w** [comment]
I'm sorry, but I'm very inexperienced with Windows unfortunately so cannot be of much help... I would recommend using something like GPT or Gemini to convert the `train.sh` script into a powershell compatible script, so that you can launch training with that.

**Comment by MaulanaYusufIkhsanRobbani** [comment]
That's okay. Perhaps I need to use python command instead.
